### ISSUE TITLE
Allow configuration of MethodLength to permit multi-line hashes

### ISSUE THREAD

**Issue by baxter** [rubocop/rubocop#1382]
If I have a file that looks like this

``` ruby
class Foo
  def long_method
    do_something
    do_something
    do_something
    do_something
    do_something
    do_something
    do_something
    do_something
    do_something
    do_something
    do_something
  end

  def single_hash_method
    {
      a: 1,
      b: 2,
      c: 3,
      d: 4,
      e: 5,
      f: 6,
      g: 7,
      h: 8,
      i: 9,
      j: 10
    }
  end
end
```

The method `long_method` is correctly identified as exceeding the permitted method length.

The method `single_hash_method` is also identified as exceeding the permitted method length.

I want to be able to configure RuboCop so that `long_method` is marked as violating MethodLength, but `single_hash_method` is permitted (the reason being that a multi-line hash or array is not complex).

Alternatively, I would be happy if I could disable MethodLength and there were another cop that I could use that would mark `long_method` as invalid but `single_hash_method` as valid.

My apologies if this is already possible or if this has previously been rejected.
